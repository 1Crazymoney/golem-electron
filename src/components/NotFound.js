import React, { Component } from 'react'

/**
 * { Class for not found(404) component. }
 *
 * @class      NotFound (name)
 */
export default class NotFound extends Component {

    render() {
        return (
            <div className="nf__container">
              <h3>The section you're looking for is ..mmh.. not here.<br/><span className="nf__sad"> :( </span></h3>
            </div>
        );
    }
}