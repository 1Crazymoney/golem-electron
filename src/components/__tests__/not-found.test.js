jest.unmock('../NotFound')

import React from 'react'
import ReactDOM from 'react-dom'
import TestUtils from 'react-addons-test-utils'
import NotFound from '../NotFound'


it('should render a notfound component', () => {
    const wrapper = shallow(
        <NotFound/>
    )
    expect(wrapper).toMatchSnapshot()
})