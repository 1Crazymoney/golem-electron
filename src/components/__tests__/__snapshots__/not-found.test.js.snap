exports[`test should render a notfound component 1`] = `
<div
  className="nf-container">
  <h3>
    The section you\'re looking for is ..mmh.. not here.
    <br />
    <span
      className="nf-sad">
       :( 
    </span>
  </h3>
</div>
`;
